lms/static/lms/js/spec/preview/preview_factory_spec.js

```diff
@@ -70,7 +70,6 @@ define(
                 var requests = AjaxHelpers.requests(this),
                     reloadSpy = spyOn(ViewUtils, 'reload');
                 showPreview({
-                    cohortedUserPartitionId: 'test_partition_id',
```

Comment:
This is no longer needed since the partition_id comes in from the select option.